src/app/common/hooks/use-get-virtuoso-height.ts

```diff
@@ -0,0 +1,36 @@
+import { token } from 'leather-styles/tokens';
+
+import { pxStringToNumber } from '@shared/utils/px-string-to-number';
+
+import { useViewportMinWidth } from '@app/common/hooks/use-media-query';
+import { useHasLedgerKeys } from '@app/store/ledger/ledger.selectors';
+import { getHeightOffset } from '@app/ui/components/containers/dialog/dialog';
+
+// virtuosoHeight = calc(InteractiveItem height - negative margin)
+// calc(72px - 24px) = 58
+const virtuosoHeight = 58;
+
+type VirtuosoVariants = 'footer' | 'no-footer' | 'popup';
+
+export function useGetVirtuosoHeight(accountNum: number, variant: VirtuosoVariants) {
```

Comment:
This function is named/located as thought it's a generic virtuoso list height function. But it isn't, because the first argument is specific to the accounts list, one particular instance of it.

Isn't this a coupling of unrelated concerns?

Reply:
This is still inflight but when needed I can refactor `accountNum` to be data array. 

We use `Virtuoso` in 3 areas and all of them relate to rendering of accounts lists

Reply:
> We use Virtuoso in 3 areas and all of them relate to rendering of accounts lists

If you'll allow me to rebuttal this: the fact that the virtual lists we use currently relate to account lists, isn't reason enough to presume all future virtual lists are going to relate to account lists. 

This code (if only conceptually) couples together a generic virtual list function with a specific type of data (accounts). I think that's a dangerous assumption, and one that breaks the single responsibility principal.

I imagine this situation: another developer is task with building an expensive list to render. They need to use this virtuoso function to get the height, and end up being hampered/confused by the fact there's account-specific in there, that they don't need or want.

What are your thoughts on this?

Reply:
Perhaps I didn't communicate effectively previously but I will be changing this code and updating the variable names. The work is not finished yet and I'm unsure if this code will be needed, ideally not. That's why I posted it as a draft. I am trying to first fix the bug, then refactor it to be better.

I agree totally with you and I don't want to break the single responsibility principle. It doesn't make sense to couple this to accounts at all. At this point its just the name I was using to as the data is accounts and I am focusing on the UI bugs. 

Once thats fixed I will update the code to make it clean and reusable for the future